specs/merge/beacon-chain.md

```diff
@@ -59,148 +58,138 @@ We define the following Python custom types for type hinting and readability:
 | Name | Value |
 | - | - |
 | `MAX_BYTES_PER_OPAQUE_TRANSACTION` | `uint64(2**20)` (= 1,048,576) |
-| `MAX_EXECUTION_TRANSACTIONS` | `uint64(2**14)` (= 16,384) |
+| `MAX_TRANSACTIONS_PER_PAYLOAD` | `uint64(2**14)` (= 16,384) |
 | `BYTES_PER_LOGS_BLOOM` | `uint64(2**8)` (= 256) |
 
 ## Configuration
 
-Warning: this configuration is not definitive.
+### Merge
 
-### Transition
+*Note*: The configuration value `MERGE_FORK_EPOCH` is not final.
 
 | Name | Value |
 | - | - |
 | `MERGE_FORK_VERSION` | `Version('0x02000000')` |
-| `MERGE_FORK_EPOCH` | `Epoch(18446744073709551615)` **TBD** |
-| `TRANSITION_TOTAL_DIFFICULTY` | **TBD** |
```

Comment:
This constant is used in [fork-choice.md](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/merge/fork-choice.md#transitionstore)

Reply:
Absolutely—on my todo to put in fork-choice.md (as stated in the PR description). Resolving for now this is a draft PR focused on beacon-chain.md only. Will fix everything once we have consensus on beacon-chain.md.